packages/ui-app/src/InputStorage/index.tsx

```diff
@@ -44,10 +44,11 @@ class InputStorage extends React.PureComponent<Props, State> {
     super(props);
 
     const { section } = this.props.defaultValue;
+    const type = 'public';
```

Comment:
Is there a reason this is actually passed through? (storage always being public - really don't need to define it explicitly as a variable?)

Reply:
I just noticed it was done this way here https://github.com/polkadot-js/apps/blob/master/packages/ui-app/src/InputExtrinsic/index.tsx#L54 where it's also passed through